### PR TITLE
TRIAGE-3541, partially read properties for non-resident reparse point.

### PR DESCRIPTION
**Pull request by Tony-Nuix**

### CODE REVIEW

File: src/main/java/org/jnode/fs/ntfs/StandardInformationAttribute.java [hunk 1]
```diff
@@ -148,7 +148,10 @@ public int getUpdateSequenceNumber() {
     }
 
     /**
-     * The file attribute flags.
+     * NTFS File attribute flags.
+     * <p>
+     * <a href="https://github.com/libyal/libfsntfs/blob/main/documentation/New%20Technologies%20File%20System%20(NTFS).asciidoc#file_attribute_flags">NTFS File attribute flags</a>
```
Comment (CoffeeForOven) [inline src/main/java/org/jnode/fs/ntfs/StandardInformationAttribute.java]: Is this a `@see` section like in `BootRecord`?

File: pom.xml [hunk 2]
```diff
@@ -54,4 +59,21 @@
         <slf4jVersion>1.7.32</slf4jVersion>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <executions>
```
Comment (CoffeeForOven) [inline pom.xml:68]: I'm getting a warning in IDEA about missing version, but good to see this here 👍

File: src/test/java/org/jnode/fs/ntfs/TestNtfsReparsePoint.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Tests for reparse point data in {@link NTFSAttribute}.
+ */
+public class TestNtfsReparsePoint {
+
+    @Test
+    public void testResidentReparsePoint()  {
+        byte[] data = intsToByteArray(
+                "-64 0 0 0 -56 1 0 0 0 0 0 0 0 0 3 0 -84 1 0 0 24 0 0 0 3 0 0 -96 -92 1 0 0 0 0 -48 0 -46 0 " +
```
Comment (CoffeeForOven) [inline src/test/java/org/jnode/fs/ntfs/TestNtfsReparsePoint.java]: Maybe we can save this as binary test data?

Comment (Tony-Nuix) [reply src/test/java/org/jnode/fs/ntfs/TestNtfsReparsePoint.java]: I learned it from the other test classes in the same package, they all just put a byte array like this.

**Pull request merged**